server/gitrest/src/externalStorageManager.ts

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import Axios from "axios";
+import * as winston from "winston";
+
+/**
+ * Manages api calls to external storage
+ */
+export class ExternalStorageManager {
+    constructor(private endpoint: string) {
+    }
+
+    public async readAndSync(tenantId: string, documentId: string): Promise<void> {
+        if (!process.env.EXTERNAL_STORAGE_ENABLED || process.env.EXTERNAL_STORAGE_ENABLED == "false") {
```

Comment:
`process.env.EXTERNAL_STORAGE_ENABLED != "true"` so that `process.env.EXTERNAL_STORAGE_ENABLED=no` wouldn't resolve as enabled?